src/cli.ts

```diff
@@ -31,11 +32,69 @@ const cliOptions = z
 		version: flag,
 		noColor: flag,
 		info: flag,
+		outdir: z
+			.string()
+			.optional()
+			.default(join('.vercel', 'output', 'static'))
+			.transform(path => normalizePath(resolve(path)))
+			.refine(
+				path =>
+					path.startsWith(normalizePath(resolve())) &&
+					path !== normalizePath(resolve()),
```

Comment:
```suggestion
				path => {
					const currentWorkingDirectory = normalizePath(resolve());
					return (
						path.startsWith(currentWorkingDirectory) &&
						path !== currentWorkingDirectory
					);
				},
```

Also the check could be made using a regex, but I suspect you didn't do that for readability, which I'm ok with 🙂

Reply:
also is it a necessary limitation that the output dir needs to be inside the current working dir? couldn't the output dir be anywhere?

Reply:
> also is it a necessary limitation that the output dir needs to be inside the current working dir? couldn't the output dir be anywhere?

good question. i suppose, yeah, would rather i change it to accomodate that?

Reply:
well since we already offer the flexibility to have move the built output I guess we can make it as flexible as we can 🤷, basically I don't see any downside in allowing the user to put any possible location, do you?

(also I am curious about @GregBrimble's outdir usecase, maybe he could need to be able to have the outdir anywhere?)

Reply:
> basically I don't see any downside in allowing the user to put any possible location, do you?

my reasoning was kinda just, well, i don't want to let the user accidentally set it to their home directory and wipe all their files... so yeah. im happy to change it if y'all need it to be more flexible though.

Reply:
ah yeah that sounds like a valid point... whatever's fine by me, let's see what greg thinks about it 🙂

Reply:
@GregBrimble any preference here?

Reply:
No preference. Happy to go with the more restricted option now and we can widen later if need be.